config/routes.yaml

```diff
@@ -3,5 +3,46 @@
 #    controller: App\Controller\DefaultController::index
 
 app_hello:
-    path: /hello
-    controller: App\Controller\HelloController::index
+    path: /
+    controller: App\Controller\AuthController::login
+
+app_registration:
+    path: /register
+    controller: App\Controller\RegistrationController::index
+
+app_login:
+    path: /login
+    controller: App\Controller\AuthController::login
+
+app_index:
+    path: /index
+    controller: App\Controller\IndexController::index
+
+app_logout:
+    path: /logout
+    controller: App\Controller\AuthController\logout
+
+blog_new:
+    path: /blog/new
+    controller: App\Controller\BlogController::new
+
+blog_list:
+    path: /blogs
+    controller: App\Controller\BlogController::list
+
+my_blog_list:
+    path: /myblogs
+    controller: App\Controller\BlogController::myBlogs
+
+blog_show:
+    path: /blogs/{id}
+    controller: App\Controller\BlogController::show
+
+blog_edit:
+    path: /blog/edit/{id}
+    controller: App\Controller\HandleBlogController::edit
+
+blog_delete:
+    path: /blog/delete/{id}
+    controller: App\Controller\HandleBlogController::delete
+    methods: [POST]
```

Comment:
follow the rest api standards for path and define methods in all routes

Reply:
done